### ISSUE TITLE
cache results and instantly write to clipboard when using c/C in shell mode

### ISSUE THREAD

**Issue by wis** [chubin/cheat.sh#137]
it looks like in shell mode, the the c command takes the same time as querying, is it because it's re-fetching because there's no caching?

**Comment by chubin** [comment]
What is `the c command`?

**Comment by wis** [comment]
``` 
    $ cht.sh --shell
    cht.sh> go reverse a list
    cht.sh> c
```
too bad there's no client side caching, latency is high for some people, and it can't break the flow.
do you accepts PRs that you might put in as much work refactoring as implementing it yourself?

**Comment by chubin** [comment]
@wis You are right, it is a trivial thing to fix and should be implemented anyway.
And regarding the PRs, the answer is yes.
